src/api/type-defs/inputs/UpdateProjectInput.js

```diff
@@ -0,0 +1,6 @@
+export default `
+  input UpdateProjectInput {
+    name: String
+    description: String
+  }
```

Comment:
Is the `updateProject` mutation resolver just used to update the name and description of the project? Is it being used in the frontend?